content/english/python-basics/data-structures/lists.md

```diff
@@ -4,82 +4,87 @@ draft: false
+Lists are the ordered collection of items and can be written as comma separated values inside square bracket `[]`. As we saw in our loops lesson, lists can be of any data type, for example strings or integers. 
 
 ```python
 #Creating a list of 5 fruits
 fruits = ['orange', 'apple', 'banana', 'apple', 'mango']
+```
 
-#Print element at index 2
-print(fruits[2])     ##it prints banana
+If you want to refer to a specific element of a list, you can do so by using indexes (position) inside square bracket. The first element is index (position) `0` and the list goes up to the `(number of elements in the list - 1)`. So, a list having 6 elements will have indices from 0 to 5. 
 
+```python
+#Creating a list of 5 fruits
+fruits = ['orange', 'apple', 'banana', 'apple', 'mango']
 
+#Print element at index 2
+print(fruits[2])     ##it prints banana
 ```
 
-![](listDiagram.png)
+**Index** | **Item**
+-------|-------
+0 | orange
+1 | apple
+2 | banana
+3 | apple
+4 | mango
 
 {{% notice note %}}
 
-Lists can have duplicate elements.
+As you see in this example, lists can have duplicate elements.
 
 {{% /notice %}}
 
+In python, lists do not have to all be of the same data type, so a single list can contain a combination of strings and numbers.
+
 ```python
 #lists can also have different datatypes
 my_list = ['candy', 10, 2.5]
-
 ```
 
 ### Length of the list
 
 You can get the length of the list by using `len(list)` function and the list name is passed as the parameter.
 
 ```python
-
-length = len(fruits)  ##Now length = 5
-
-
+length = len(fruits)  ##length = 5
 ```
 
 ### Append an element in the list
 
-Appending an element can be done by the `list.append(x)` function where `x` is the element that you want to append. It adds the element at the end of the list.
+You can add items to a list by using the `append` function, `list.append(x)` function where `list` is the name of the list and `x` is the element that you want to append. It adds the element at the end of the list.
 
 ```python
-
 #Before the list is ['orange', 'apple', 'banana', 'apple', 'mango']
 fruits.append('lemon')
 #Now the list is ['orange', 'apple', 'banana', 'apple', 'mango', 'lemon']
-
 ```
 
 ### Insert an element at a specific index
 
-You can insert an element at specific index by of a list by `list.insert(i, x)` function. Here `i` is the index where you want to insert and `x` is the element that you want insert.
+If you want to control where you place a new element, you can insert an element at specific index by of a list by `list.insert(i, x)` function. Here `list` is the name of the list, `i` is the index where you want to insert and `x` is the element that you want insert. Note: this will shift all of the elements starting at position i over one index.
```

Comment:
```suggestion
If you want to control where you place a new element, you can insert an element at specific index of a list by using the `list.insert(i, x)` function. Here `list` is the name of the list, `i` is the index where you want to insert and `x` is the element that you want insert. Note: this will shift all of the elements starting at position `i` over one index.
```